src/test/PurchaseArrayTest.java

```diff
@@ -0,0 +1,55 @@
+package test;
+
+import main.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PurchaseArrayTest {
```

Comment:
wrong approach to unit testing. 
you don't need to test jdk, it's not your responsibility )
you have to make tests for your own methods. 
for example, `getCost`, `Euro.toString`, `RoundMethod.round` etc. 
to test sorting you need to make tests for `compareTo`.